wordpress-mobile/WordPress-iOS
# [Stats Refresh] All Time Stats: format large numbers

Fixes #10501 

Well this was easier than expected. Turns out I'd used the wrong values to display. The string versions of the values are pre-formatted with commas. So I've simply replaced the number values with the string values. 🎉 !

Also, as a minor bonus, I disabled the selection on the cell header. So now if you select it, it won't turn an ugly brown color.

To test:
- On a site with a bunch of posts, views, visitors, and/or best views ever, go to Stats > Insights.
  - Tip: the Automattic Field Guide has some high number.
- Verify the numbers have commas:

<img width="400" alt="numbers" src="https://user-images.githubusercontent.com/1816888/48870281-71d17180-ed9d-11e8-9740-8e84bbe0ba14.png">
